ebuku.el

```diff
@@ -853,6 +853,49 @@ If an argument is excluded, get it from `ebuku-cache-default-args'."
       (setq tags (nconc (map-elt bookmark 'tags) tags)))
     (setq ebuku-tags (sort (seq-uniq tags) 'string-collate-lessp))))
 
+(defun ebuku-append-tag-to-bookmark ()
+  "Interactive function to append tag to bookmark."
+  ;; TODO fix an issue where it moves the cursor to the top after adding
+  (interactive)
+  (ebuku-update-tags-cache)
+  (let ((tag (completing-read "Append tag? " ebuku-tags))
+	(index (ebuku--get-index-at-point))
+	(inhibit-read-only t))
+    (ebuku--tags-add index tag)
+    (ebuku-refresh)
+    (message "Added tag to bookmark")))
+
+(defun ebuku--tags-join (tags)
+  "Internal function that joins a list of tags into a comma separated string of tags."
+  (typecase tags
+    (stringp tags)
+    (listp (mapconcat 'identity tags ","))))
+
+(defun ebuku--tags-add (index tags)
+  "Append tag to the list of tags on the index"
```

Comment:
Please label internal functions as such in docstrings: "Internal function to append ..."

Also, please reference the function parameters in the docstring with capitals: "Internal function to append a tag to TAGS for INDEX."

Reply:
(This applies to your other docstrings as well.)

Reply:
Thanks for the tip, applied this to the rest of the docstrings as well.